scripts/run-build-visit

```diff
@@ -265,29 +269,29 @@ if [ "$poodle" = true ]; then
    env PAR_COMPILER=/usr/tce/packages/mvapich2/mvapich2-2.3.6-gcc-10.3.1/bin/mpicc \
        PAR_COMPILER_CXX=/usr/tce/packages/mvapich2/mvapich2-2.3.6-gcc-10.3.1/bin/mpicxx \
        PAR_INCLUDE=-I/usr/tce/packages/mvapich2/mvapich2-2.3.6-gcc-10.3.1/include \
-       ./$build_visit_script --group ${dest_group} --required --optional --mesagl --parallel \
+       ./$build_visit_script --group ${dest_group} --required --optional --llvm --mesagl --parallel \
```

Comment:
I wonder if it would be better to have `bv_mesagl.sh` and `bv_osmesa.sh` call `bv_llvm_enable` in their own 'enable' functions? That would prevent the need for specifying `--llvm` on top of `--osmesa` or `--mesagl`.